### PR TITLE
feat(admin): add elgg news widget with latest news links from elgg.org

### PR DESCRIPTION
**Pull request by jdalsem**
elgg_news

using googles rss feed thingy

### CODE REVIEW

**Comment by hypeJunction** [comment]
Why not use webservices?

**Comment by jdalsem** [comment]
> Why not use webservices?

don't know... is it just as easy?? does elgg.org publish the webservices already? what webservice to use for blogs?

**Comment by hypeJunction** [comment]
Webservices are active on the community site. Just need to expose one function.

**Comment by hypeJunction** [comment]
Webservices are active on the community site. Just need to expose one function.

File: views/default/widgets/elgg_news/rss.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+		var wrapper_id = $wrapper.attr("id");
+		$wrapper.empty();
+		
+		var config = $wrapper.data();
+		
+		var feed_url = config.feedUrl;
+		var limit = config.limit;
+
+		$.getJSON(
+			"//ajax.googleapis.com/ajax/services/feed/load?v=1.0&num=" + limit + "&output=json_xml&q=" + encodeURIComponent(feed_url) + "&hl=en&callback=?",
```
Comment (mrclay) [inline views/default/widgets/elgg_news/rss.js]: This goes away in a few months. https://developers.google.com/feed/v1/jsondevguide

**Comment by mrclay** [comment]
Are we against adding a lib to read and cache RSS?

**Comment by mrclay** [comment]
How about boring old JSONP? https://github.com/Elgg/www.elgg.org/pull/110

**Comment by jdalsem** [comment]
no need... we could just use https://elgg.org/blog?view=json... much simpler

**Comment by mrclay** [comment]
IMO we want cacheable, accessible cross origin, and with minimal overhead for elgg.org. Ideal would be a plain JSONP-style .js file updated when blogs are. Nginx would handle HTTP headers.

**Pull request closed without merging**